src/main/java/de/qabel/desktop/DesktopClient.java

```diff
@@ -227,6 +228,7 @@ public void start(Stage stage) throws Exception {
                     startDropDaemon();
                     view = new LayoutView();
                     Parent view = this.view.getView();
+                    runtimeConfiguration.setWindow(((LayoutController)(this.view.getPresenter())).getWindow());
```

Comment:
still 2 superfluous brackets ;)
`((LayoutController)this.view.getPresenter).getWindow()`